diff --git config/connections.js config/connections.js
index 267195e..3316cc2 100644
--- config/connections.js
+++ config/connections.js
@@ -44,16 +44,16 @@ module.exports.connections = {
   postgresDb: {
     adapter: 'sails-postgresql',
     host: 'localhost',
-    user: 'toronto_api_user',
-    password: '',
+    user: 'postgres',
+    password: 'sekret',
     database: 'toronto'
   },
 
   pupaDb: {
     adapter: 'sails-postgresql',
     host: 'localhost',
-    user: 'toronto_api_user',
-    password: '',
+    user: 'postgres',
+    password: 'sekret',
     database: 'pupa'
   }
 
